dbc2val/mapping.yml

```diff
@@ -27,68 +27,55 @@
 # Note VSS is angle of the steering wheel, DBC might be angle or steering?
 # Converting to int
 SteeringAngle129:
-  vss: # taken from https://github.com/COVESA/vehicle_signal_specification/blob/master/spec/Chassis/Chassis.vspec
-    datatype: int16 
-    type: sensor
-    unit: degrees
-    description: Steering wheel angle. Positive = degrees to the left. Negative = degrees to the right.
-  databroker:
-    datatype: 3 # INT16 number taken from types_pb2.py
-    changetype: 2 # ON_CHANGE number taken from types_pb2.py
   minupdatedelay: 100
   targets:
-    Vehicle.Chassis.SteeringWheel.Angle:
+    Vehicle.Chassis.SteeringWheel.Angle:  # taken from https://github.com/COVESA/vehicle_signal_specification/blob/master/spec/Chassis/Chassis.vspec
+      vss:
+        datatype: int16
+        type: sensor
```

Comment:
why do we still need all those vss entries?

I understand, "datatype" is needed as GRPC is typed, and _maybe_ type in case we register datapoints (so db knows how to handle it), but unit and description?

Reply:
I think "unit" and "description" are useful if we want to register data points "properly".
Currently though, only "path", "datatype" and "description" are used to register missing data points.
Also, we have no clue currently which version of the VSS was used to generate those "vss" entries.
I think it could be better if that mapping.yml only maps CAN signal IDs to VSS paths and that we then rely on the VSS JSON spec to register missing data points.